### PR TITLE
Configure kibana from the environment

### PR DESCRIPTION
**Pull request by alexmuller**
Instead of using alphagov-deployment.

### CODE REVIEW

File: modules/govuk/manifests/apps/kibana.pp [hunk 1]
```diff
@@ -1,13 +1,63 @@
...
+) {
+  $app_name = 'kibana'
+
+  govuk::app { $app_name:
     app_type => 'rack',
     port     => $port,
   }
+
+  Govuk::App::Envvar {
+    app => $app_name,
```
Comment (deanwilson) [inline modules/govuk/manifests/apps/kibana.pp]: Is there a reason this is a variable when it's not possible to change it when calling the class?

File: modules/govuk/manifests/apps/kibana.pp [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
-  govuk::app { 'kibana':
+# [*signon_client_secret*]
+#   The secret which matches signon_client_id
+#
+# [*signon_root*]
+#   The HTTP location of Signon
+#
+class govuk::apps::kibana (
+  $elasticsearch_host = 'logs-elasticsearch-1.management',
+  $port = '3202',
```
Comment (deanwilson) [inline modules/govuk/manifests/apps/kibana.pp]: Ports are numbers.

**Comment by deanwilson** [comment]
A couple of small queries but it looks fine to merge.

File: modules/govuk/manifests/apps/kibana.pp (hunk 1 above)
Comment (alexmuller) [inline modules/govuk/manifests/apps/kibana.pp]: Mostly just because it's duplicated on line 36 but no other reason!

Comment (deanwilson) [inline modules/govuk/manifests/apps/kibana.pp]: Fair enough.

**Comment by deanwilson** [comment]
Looks good. Merge away.

File: modules/govuk/manifests/apps/kibana.pp [hunk 3]
```diff
@@ -1,13 +1,63 @@
 # == Class: govuk::apps::kibana
 #
-# Set up the Kibana application
+# Set up the Kibana application behind Signon
+#
+# === Parameters
+#
+# [*elasticsearch_host*]
+#   Which logs-elasticsearch host Kibana should connect to, must
+#   be a single host: https://github.com/elasticsearch/kibana/issues/214
```
Comment (rjw1) [inline modules/govuk/manifests/apps/kibana.pp]: sigh

**Comment by rjw1** [comment]
:+1:

File: modules/govuk/manifests/apps/kibana.pp (hunk 3 above)
Comment (alexmuller) [inline modules/govuk/manifests/apps/kibana.pp]: Good to see that issue hasn't changed in a couple of years.

**Pull request merged**